planetary-social/nos
# Dramatically improve the search experience

## Issues covered
#1242 

## Description
This is how I think search should work. I didn't really follow the requirements laid out in #1242 but I'd like you to try this and if you like it, I'd like Linda to try it. And yeah, we can clean up the code before she gets it, so go ahead and make suggestions. I just haven't had time to think through all of it yet, or to figure out the best way to do things.

## To Do
- [x] Fix issue with pasting note IDs in the search bar. Try: `note12n864kvrnp3vp49hzgze9lp5fnt036g65j3au8kdx3whzfjp3a7sy7v7ad`
- [x] Fix issue where pasting npub causes multiple author views to be pushed
- [ ] Consider reducing lag while typing
- [ ] Consider adding a spinner to the bottom of the list

## How to test
1. Navigate to Discover
2. Type your search term into the search bar
3. Observe

## Screenshots/Video
I'm gonna leave this blank for now so _you_ can try it yourself before seeing how _I_ use it.
